README.md

```diff
@@ -122,8 +121,9 @@ The course configuration will be located under a new folder with its correspondi
 
 To make your data available in the system use the `upload_data.py` script. This will upload your local data to a NFS server.
 
-For instance, you can do:
+> **NOTE:** You might need to grant execution privileges to the upload script.
```

Comment:
This note should not be needed, it should have that in the repo already. If it doesn't something is wrong and we should fix it there.

Reply:
We experienced in one of the tests we did. Shall we keep it for now?